.github/workflows/build-ubuntu.yml

```
@@ -105,7 +102,7 @@ jobs:
           restore-keys: |
             vcpkg-${{ matrix.buildtype }}-
       - name: Setup vcpkg
-        uses: lukka/run-vcpkg@a400452f634fe49e9f18d388aeb1809dcc642136
+        uses: lukka/run-vcpkg@${{ env.LUKKA_RUN_VCPKG_SHA }}
```

actionlint (1.7.11)

[error] context "env" is not allowed here. no context is available here. see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)